boxes/boxes/vanilla/app/embedded-wallet.ts

```diff
@@ -271,12 +271,15 @@ export class EmbeddedWallet extends BaseWallet {
       instance,
       artifact,
     } = await this.getFakeAccountDataFor(opts.from);
-    const fee = await this.getFeeOptions(
+    const { fee, gasEstimationsSimulationResults } = await this.getFeeOptions(
```

Comment:
The fetFeeOptions name is misleading as it returns what I would not expect - gasEstimationsSimulationResults